src/utils.py

```diff
@@ -194,3 +204,16 @@ async def get_repo(name: str) -> github3.repos.repo.Repository:
     github_token = await Secret.load("collection-registry-github-token")
     gh = await run_sync_in_worker_thread(github3.login, token=github_token.get())
     return gh.repository("PrefectHQ", name)
+
+
+def validate_view_content(view_dict: dict, variety: CollectionViewName) -> None:
+    """Raises an error if the view content is not valid."""
+    schema = getattr(metadata_schemas, f"{variety}_schema")
+    validate = fastjsonschema.compile(schema)
+
+    for collection_name, collection_metadata in view_dict.items():
+        try:
+            validate(list(collection_metadata.values())[0])
```

Comment:
Wondering what does this `list(collection_metadata.values())[0]` give? We're getting the first key?

Reply:
yeah it should be the one and only value for this dictionary, although it would probably make more sense to use the key explicitly if im going to use .items()

Reply:
i think i need to be more explicit about this, realizing now it should be slightly different for blocks